packages/muelu/src/Utils/MueLu_PerfUtils_decl.hpp

```diff
@@ -54,7 +54,7 @@
 #include <Xpetra_Import_fwd.hpp>
 #include <Xpetra_Matrix_fwd.hpp>
 
-//#include "MueLu_Utilities_fwd.hpp"
+#include "MueLu_PerfUtils_fwd.hpp"
 
```

Comment:
MueLu_PerfUtils.hpp is auto-generated and depends on the ETI. Thus, if we don't include it here then we'll have to include the forward file explicitly everywhere.

All MueLu _decl file should include the _fwd files, and almost all do.